lib/rspec/rails/view_rendering.rb

```diff
@@ -44,6 +44,13 @@ def render_views?
       #
       # @private
       class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+
+        def initialize(path, pattern=nil)
+          unless path.is_a?(::ActionView::Resolver)
+            super
+          end
+        end
+
```

Comment:
The problem is that when `path` is a `Resolver` instance, it knows how to look up templates. And the `super` call in this `EmptyTemplateResolver#find_templates` doesn’t call up to that resolver.